src/cli.mjs

```diff
@@ -83,6 +86,12 @@ function buildYargs(argvInput) {
       },
       builder: {},
     })
+    .command("warn", false, {
+      handler: async () => {
+        process.emitWarning("this is a warning emited on the node process");
+      },
+      builder: {},
+    })
```

Comment:
does this expose the command to the users?

Reply:
it's technically runnable, but it doesn't show up in completions, help text, etc. the second arg (false) tells yargs to hide the command.

Reply:
Instead of hidden commands in prod, could we use env vars to produce these in dev only.

And then have another flag that toggles warning emission when the NODE_ENV is not production.

Reply:
they can be added conditionally behind ENV vars - that's a great idea. these commands are primarily for the tests to exercise certain failures in a controllable ay.

this change _does_ set warning to only emit when NODE_ENV is production.

Reply:
cool. mind making that change?

Reply:
done!